Build/Unicorn.Roles.nuget/Unicorn.Roles.nuspec

```diff
@@ -12,15 +12,25 @@
 		<description>Adds security role syncing to Unicorn. This package contains both the core library and the configuration for it, which is appropriate for web projects. Install Unicorn.Roles.Core if you only want the library.</description>
 		<copyright>Copyright 2018</copyright>
 		<tags>sitecore serialization</tags>
-	<dependencies>
-		<dependency id="Unicorn" version="$version$" />
-		<dependency id="Unicorn.Roles.Core" version="$version$" />
-	</dependencies>
+		<dependencies>
+			<dependency id="Unicorn" version="$version$" />
+			<dependency id="Unicorn.Roles.Core" version="$version$" />
+		</dependencies>
+		<!-- PackageReference support -->
+		<contentFiles>
+			<files include="any/**/*" buildAction="Content" copyToOutput="true" />
+		</contentFiles>
 	</metadata>
 	<files>
+		<!-- PackageReference support -->
+		<file src="..\..\src\Unicorn.Roles\Standard Config Files\Unicorn.Roles.config" target="contentFiles\any\any\App_Config\Include\Unicorn" />
```

Comment:
Can I suggest that configs get installed to App_Config\Modules\Unicorn instead? This follows the change in SXA 1.8 which installs SXA to App_Config\Modules\SXA.

Reply:
This will break compatibility with Sitecore 8. The only reason SXA is able to do it in 1.8 is because it only supports Sitecore 9+. Though I agree with you 100%--I'd love to see this change.

Reply:
Perhaps we can just update the readme then

Reply:
Or could create a new nuget package called Unicorn.Config for Sitecore 9+?